src/App.css

```diff
@@ -1502,3 +1502,17 @@ a:active {color:#213960;}  /* selected link */
 .row:after,
 .form-horizontal .form-group:after {
   clear: both; }
+
```

Comment:
Code looks good, copy and paste it up in the articles section to keep the custom stylings a little more organized and then recommit.

Reply:
fixed and recommitted